From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Engberg <daniel.engberg.lists@pyret.net>
Date: Mon, 30 Apr 2018 08:11:13 +0200
Subject: [PATCH] shared/util: include endian.h

util.h uses the __BYTE_ORDER macros, which musl declares in endian.h
instead of pulling them in through sys/types.h.
---
 src/shared/util.h | 1 +
 1 file changed, 1 insertion(+)

--- a/src/shared/util.h
+++ b/src/shared/util.h
@@ -20,6 +20,7 @@
 #include <sys/types.h>
 #include <sys/uio.h>
 #include <unistd.h>
+#include <endian.h>
 
 #define ARRAY_SIZE(arr) (sizeof(arr) / sizeof((arr)[0]))
 #define BIT(n)  (1 << (n))
